<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Script_Editor_DockWidget</class>
 <widget class="QDockWidget" name="Script_Editor_DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script Editor</string>
  </property>
  <widget class="QWidget" name="Script_Editor_dockWidgetContents">
   <layout class="QGridLayout" name="Script_Editor_dockWidgetContents_gridLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="menuBar_frame">
      <layout class="QGridLayout" name="menuBar_frame_gridLayout">
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="Script_Editor_splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPlainTextEdit" name="Script_Editor_Output_plainTextEdit"/>
      <widget class="QWidget" name="gridLayoutWidget">
       <layout class="QGridLayout" name="Script_Editor_gridLayout">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="Script_Editor_Input_plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="Buttons_horizontalLayout">
      <item>
       <widget class="QPushButton" name="Evaluate_Selection_pushButton">
        <property name="statusTip">
         <string>Evaluate Selection Push Button: Evaluates selected &quot;Script_Editor_Input_textEdit&quot; QTextEdit content as Python code.</string>
        </property>
        <property name="text">
         <string>Evaluate Selection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Evaluate_Script_pushButton">
        <property name="statusTip">
         <string>Evaluate Script Push Button: Evaluates &quot;Script_Editor_Input_textEdit&quot; QTextEdit content as Python code.</string>
        </property>
        <property name="text">
         <string>Evaluate Script</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
